Form/DoctrineMongoDBTypeGuesser.php

```diff
@@ -121,13 +116,8 @@ public function guessType($class, $property)
         }
     }
 
-    /**
-     * @param string $class
-     * @param string $property
-     *
-     * @return ValueGuess|null
-     */
-    public function guessRequired($class, $property)
+    /** @return ValueGuess|null */
+    public function guessRequired(string $class, string $property)
```

Comment:
[`FormTypeGuesserInterface::guessRequired()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Form/FormTypeGuesserInterface.php#L31)